commit 653a31f17beb7a131c0b563f1e2df5c8bb1a56a2
Author: KT Liao <kt.liao@emc.com.tw>
Date:   Mon Aug 14 20:11:59 2017 -0700

    Input: elan_i2c - Add antoher Lenovo ACPI ID for upcoming Lenovo NB
    
    commit 76988690402dde2880bfe06ecccf381d48ba8e1c upstream.
    
    Add 2 new IDs (ELAN0609 and ELAN060B) to the list of ACPI IDs that should
    be handled by the driver.
    
    Signed-off-by: KT Liao <kt.liao@emc.com.tw>
    Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1638_653a31f17beb7a131c0b563f1e2df5c8bb1a56a2.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1638_653a31f17beb7a131c0b563f1e2df5c8bb1a56a2.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
